wormchain/README.md

```diff
@@ -16,18 +16,18 @@ See [development.md](./development.md)
 
 ## How to run the tests
 
-    run either "starport chain serve" or "tilt up"
+    run "tilt up"
```

Comment:
I think tilt won't run wormchain by default so this probably should be `tilt up -- --wormchain`.